### PR TITLE
Update GMSmyloid5.2 and add new small lymphoma

### PR DESCRIPTION
**Pull request by annaengstrom**
Check list if changes affect bed files:

- [x] Update shasum
- [x] Update annotations
- [x] Update filename if relevant
- [x] Update person who designed/changed it
- [x] Update bed file version
- [x] Update UCSC track

### CODE REVIEW

**Comment by annaengstrom** [comment]
Now I realized that I made two "major" changes in one pull request - this is maybe not common practice?

**Comment by hassanfa** [comment]
I think you forgot to add bedfiles. You changed the version in the readme but there are no bedfiles to accompany them.  Myeloid 5.2 bed file is on another branch that I pushed a while ago, but I think I can merge that into this one.

That'll leave us with lymphoid panel.

**Comment by hassanfa** [comment]
> Now I realized that I made two "major" changes in one pull request - this is maybe not common practice?

Don't worry about major/minor/patch changes in PR, change is good.

**Comment by annaengstrom** [comment]
Think I did now what you asked for, please check :)

File: data_files/Lymphoma_hg19.bed [hunk 1]
```diff
@@ -0,0 +1,461 @@
+browser position chr1:2487773-2488202
+track name=Covered_TE-41085903 description=“COVERED-TE-41085903” color=0,128,0 db=hg19
```
Comment (hassanfa) [inline data_files/Lymphoma_hg19.bed]:
```suggestion
track name=GMSlymphoma description=“GMSlymphoma panel” color=0,128,0 db=hg19
```

File: data_files/Lymphoma_hg19.bed [hunk 2]
```diff
@@ -0,0 +1,461 @@
+browser position chr1:2487773-2488202
+track name=Covered_TE-41085903 description=“COVERED-TE-41085903” color=0,128,0 db=hg19
+##Wirta_Lymphoma_TE-41085903_hg19
```
Comment (hassanfa) [inline data_files/Lymphoma_hg19.bed]:
```suggestion
## GMSlymphoma Gene Panel Clinical Genomics
```

**Comment by hassanfa** [comment]
Now the missing part is UCSC, changelog, and versionbump. I'll take care of those.

**Comment by annaengstrom** [comment]
> Now the missing part is UCSC, changelog, and versionbump. I'll take care of those.

:+1:

This was fun! Bring on more!

**Pull request merged**